reg_tests/test_files/oversetMovingCylinder/oversetMovingCylinder.yaml

```diff
@@ -0,0 +1,232 @@
+Simulations:
+  - name: sim1
+    time_integrator: ti_1
+    optimizer: opt1
+
+linear_solvers:
+  - name: solve_scalar
+    type: tpetra
+    method: gmres
+    preconditioner: sgs
+    tolerance: 1e-5
+    max_iterations: 50
+    kspace: 50
+    output_level: 0
+
+  # TODO -TPETRA Solver is failing with this test
```

Comment:
FYI @jhux2. This may be something simple but I haven't had a chance to debug it.

Reply:
Looks like all the overset cases using Muelu for continuity are failing for me on my mac.  Not sure what's going on with that. @jrood-nrel can you check on you mac and see if you get the same thing